P5/Source/Specs/encodingDesc.xml

```diff
@@ -15,44 +15,38 @@ source or sources from which it was derived.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">전자 텍스트와 그것의 원전 텍스트 혹은 텍스트들 사이의 관련성을 기록한다.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">記錄電子文件和其ㄧ個或多個文件來源之間的關係。</desc>
   <desc versionDate="2008-04-05" xml:lang="ja">電子テキストとその元資料との関係を示す。</desc>
-    <desc versionDate="2016-11-17" xml:lang="de">dokumentiert das Verhältnis zwischen dem elektronischen Text und seiner Quelle oder den Quellen, von denen er sich ableitet.</desc>
+  <desc versionDate="2016-11-17" xml:lang="de">dokumentiert das Verhältnis zwischen dem elektronischen Text und seiner Quelle oder den Quellen, von denen er sich ableitet.</desc>
   <desc versionDate="2007-05-04" xml:lang="es">documenta la relación entre un texto electrónico y la fuente o fuentes de las que este deriva.</desc>
   <desc versionDate="2007-01-21" xml:lang="it">documenta la relazione fra un documento elettronico e la o le fonti da cui è derivato.</desc>
   <classes>
     <memberOf key="att.global"/>
     <memberOf key="model.teiHeaderPart"/>
   </classes>
   <content>
-    
-      
-        <alternate minOccurs="1" maxOccurs="unbounded">
-          <classRef key="model.encodingDescPart"/>
-          <classRef key="model.pLike"/>
-        </alternate>
-      
-    
+    <alternate minOccurs="1" maxOccurs="unbounded">
+      <classRef key="model.encodingDescPart"/>
+      <classRef key="model.pLike"/>
+    </alternate>
```

Comment:
I would suggest interleaving a reference to an optional constraintDecl here.

```xml
<sequence preserveOrder="false">
      <alternate minOccurs="1" maxOccurs="unbounded">
       <classRef key="model.encodingDescPart"/>
        <classRef key="model.pLike"/>
      </alternate>
      <elementRef key="constraintDecl" minOccurs="0" maxOccurs="1"/>
    </sequence>
```

Reply:
I think this is no longer relevant if we're making constraintDecl a member of model.encodingDescPart